romana-install/roles/romana/install-agent/templates/config/romana.yaml

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Ok, but you have curly braces here and clearly JSON syntax. The content is not YAML, but JSON.

This will not be parsed successfully by a YAML parser.